src/shared/api/urls.ts

```diff
@@ -207,7 +207,3 @@ export function getDocsUrl(sourceUrl:string,docsBaseUrl?:string): string {
         return docsBaseUrl + "/" + sourceUrl;
     }
 }
-
-export function getLegacyDataAccessTokenUrl(){
```

Comment:
maybe this function is no longer needed? At least the reference to api-legacy in the endpoint seems out of date now. If the function is still needed, maybe the name should just be "getDataAccessTokenUrl()"

Reply:
@sheridancbio yes, I removed it now that it's not needed anymore